common/i18n.py

```
@@ -0,0 +1,423 @@
+    user_lang = None
+    try:
+        from quart import g, request
+
+        header = request.headers.get("Accept-Language")
+        user = getattr(g, "user", None)
+        if user is not None:
+            user_lang = getattr(user, "language", None)
+        if user_lang or header:
+            return resolve_locale(header, user_lang, os.environ.get("LANG"))
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)